RELEASE_NOTES.md

```diff
@@ -1,5 +1,10 @@
 # Release Notes
 
+## 1.36 18-Nov-2019
+
+Added mechanism to add configuration file-based custom reorganization applet at compilation time.
```

Comment:
How about:

Added a mechanism for a custom reorganization applet, it can be used instead of the built in `--reorg` option. You can use it to reorganize workflow file results after it completes.